src/SdfGenerator_TEST.cc

```diff
@@ -135,16 +135,43 @@ TEST(CompareElements, CompareWithDuplicateElements)
   EXPECT_FALSE(isSubset(m1CompTest, m1));
 }
 
+
+const char *g_cacheLocation = nullptr;
```

Comment:
Thanks for the suggestion. I created a helper class and renamed it to `UniqueTestDirectoryEnv` since it can be used anytime you need a test directory.
https://github.com/ignitionrobotics/ign-gazebo/pull/213/commits/35281a7e2a79929f0495d7c6483931efb360807f